docs/bootstrap/main.tf

```diff
@@ -10,25 +16,25 @@ variable "ssh_key_file" {
 }
 
 provider "digitalocean" {
-  token = var.do_token	
+  token = var.do_token
 }
 
 resource "random_password" "password" {
-  length = 16
-  special = true
+  length           = 16
+  special          = true
   override_special = "_-#"
 }
 
-data "local_file" "ssh_key"{
-  filename = pathexpand(var.ssh_key_file)
+data "template_file" "cloud_init" {
+  template = file("cloud-config.tpl")
+  vars = {
+    gw_password = random_password.password.result
+  }
 }
 
-data "template_file" "cloud_init" {
```

Comment:
What happened to this?

Reply:
hmm nothing its still there :)

https://github.com/dirien/faasd/blob/8db224d572a2faae2eab75d8326c727b27364d29/docs/bootstrap/digitalocean-terraform/main.tf#L47

https://github.com/dirien/faasd/blob/8db224d572a2faae2eab75d8326c727b27364d29/docs/bootstrap/main.tf#L28

Reply:
@alexellis